go.mod

```diff
@@ -194,6 +194,8 @@ replace (
 	// use cosmos fork of keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 	github.com/cometbft/cometbft-db => github.com/crypto-org-chain/cometbft-db v0.0.0-20230412133340-ac70df4b45f6
+	// https://github.com/cosmos/cosmos-sdk/pull/16331
```

Comment:
do we include these feature in this [diff](https://github.com/cosmos/cosmos-sdk/compare/v0.46.3...crypto-org-chain:cosmos-sdk:885b47bfbe64)